image.yaml

```diff
@@ -79,7 +79,7 @@ modules:
           - name: os-eap-hawkular
           - name: os-eap-deployment-scanner
           - name: os-eap-extensions
-          - name: os-sso73
+          - name: os-sso72
```

Comment:
Why this was moved back? Should we create the ```os-sso73``` one module? I can do that in the adapters PR.